File: 30.2/ubuntu/26.04/aarch64/systems-ide/flight-tests/ruby/deploy.rb
# Flight-test for systems-ide's Ruby glue-script tier.
#
# Mirrors a Chef-recipe-style deployment script: plain Ruby only, no chef
# gem dependency, matching this tier's no-project/no-dependency-manager
# scope.

require_relative "helpers"

PACKAGES = %w[nginx app].freeze

puts "hostname: #{Helpers.run('hostname')}"
PACKAGES.each { |pkg| Helpers.ensure_package(pkg) }

File: 30.2/ubuntu/26.04/aarch64/systems-ide/flight-tests/python/deploy.py
"""Flight-test for systems-ide's Python glue-script tier.

Mirrors a one-off Fabric-style deployment script: plain stdlib only, no
fabric/paramiko dependency, matching this tier's no-project/no-
dependency-manager scope.
"""

import tasks

SERVICES = ["nginx", "app"]


def main() -> None:
    print(f"hostname: {tasks.run('hostname')}")
    for service in SERVICES:
        tasks.restart_service(service)


if __name__ == "__main__":
    main()
